src/TransactionalSession.CosmosDB.AcceptanceTests/TransactionalSession.CosmosDB.AcceptanceTests.csproj

```
@@ -28,7 +30,11 @@
   <Import Project="..\$(TemplateProject)\Includes.targets" />
 
   <ItemGroup>
-    <Compile Include="..\$(TemplateProject)\**\*.cs" Exclude="\**\obj\**\*.*;"/>
+    <Compile Include="..\$(TemplateProject)\**\*.cs" Exclude="\**\obj\**\*.*;" />
```

```suggestion
    <Compile Include="..\$(TemplateProject)\**\*.cs" Exclude="..\$(TemplateProject)\obj\**\*.*;" />
```
Would this work? (and negate the need for the other changes to the project file)